boards/px4/io-v2/nuttx-config/nsh/defconfig

```diff
@@ -65,6 +65,6 @@ CONFIG_USART3_RXBUFSIZE=64
 CONFIG_USART3_RXDMA=y
 CONFIG_USART3_TXBUFSIZE=64
 CONFIG_USEC_PER_TICK=1000
-CONFIG_USERMAIN_STACKSIZE=1132
+CONFIG_USERMAIN_STACKSIZE=1056
```

Comment:
```suggestion
CONFIG_USERMAIN_STACKSIZE=1100
```

This leaves 12 words.

At 1056 rebooting the IO then the FMU would blow through the stake and corrupt the heap. 


```
nsh>  px4io status
WARN  [px4io] loaded
protocol 4 hardware 2 bootloader 3 buffer 64B crc 0x21f80d51
8 controls 8 actuators 18 R/C inputs 2 analog inputs 0 relays
56171 bytes free
status 0x2765 OUTPUTS_ARMED SAFETY_SAFE RC_OK SBUS FMU_OK MIXER_OK ARM_SYNC INIT_OK
alarms 0x0010 FMU_LOST
vservo 19 mV vservo scale 10000
vrssi 0
actuators -9914 -10000 -10000 0 10000 -9965 0 0
servos 900 900 900 900 0 0 0 0
reversed outputs: [________] trims: r:   0.0000 p:   0.0000 y:   0.0000
18 raw R/C inputs 1498 1498 1861 1489 1418 1003 1494 982 1494 1494 1494 1494 1494 1494 1494 1494 998 998
R/C flags: 0x0010
mapped R/C inputs 0x0000
ADC inputs 2 0
features 0x0001 S.BUS1_OUT
arming 0x0861 FMU_DISARMED FMU_NOT_PREARMED IO_ARM_OK INAIR_RESTART_OK ALWAYS_PWM_ENABLE
rates 0x000f default 50 alt 400 sbus 72
debuglevel 0
controls 0: 11 98 44 0 0 0 0 0
controls 1: 0 0 0 0 0 0 0 0
controls 2: 0 0 0 0 0 0 0 0
controls 3: 0 0 0 0 0 0 0 0
input 0 min 1000 center 1500 max 2000 deadzone 10 assigned 255 options 0x0000
input 1 min 1000 center 1500 max 2000 deadzone 10 assigned 255 options 0x0000
input 2 min 1000 center 1500 max 2000 deadzone 10 assigned 255 options 0x0000
input 3 min 1000 center 1500 max 2000 deadzone 10 assigned 255 options 0x0000
input 4 min 1000 center 1500 max 2000 deadzone 10 assigned 255 options 0x0000
input 5 min 1000 center 1500 max 2000 deadzone 10 assigned 255 options 0x0000
input 6 min 1000 center 1500 max 2000 deadzone 10 assigned 255 options 0x0000
input 7 min 1000 center 1500 max 2000 deadzone 10 assigned 255 options 0x0000
input 8 min 1000 center 1500 max 2000 deadzone 0 assigned 255 options 0x0000
input 9 min 1000 center 1500 max 2000 deadzone 0 assigned 255 options 0x0000
input 10 min 1000 center 1500 max 2000 deadzone 0 assigned 255 options 0x0000
input 11 min 1000 center 1500 max 2000 deadzone 0 assigned 255 options 0x0000
input 12 min 1000 center 1500 max 2000 deadzone 0 assigned 255 options 0x0000
input 13 min 1000 center 1500 max 2000 deadzone 0 assigned 255 options 0x0000
input 14 min 1000 center 1500 max 2000 deadzone 0 assigned 255 options 0x0000
input 15 min 1000 center 1500 max 2000 deadzone 0 assigned 255 options 0x0000
input 16 min 1000 center 1500 max 2000 deadzone 0 assigned 255 options 0x0000
input 17 min 1000 center 1500 max 2000 deadzone 0 assigned 255 options 0x0000
failsafe 900 900 900 0 2100 900 1500 1500
disarmed values 900 900 900 900 0 0 0 0
```
With 1100
![image](https://user-images.githubusercontent.com/1945821/79278636-ef00f900-7e60-11ea-829c-da3a26eb8581.png)